### PR TITLE
feat: Configurable visibility in whitelabel.yml

### PR DESCRIPTION
**Pull request by spydon**

### CODE REVIEW

File: packages/ubuntu_bootstrap/lib/installer.dart [hunk 1]
```diff
@@ -108,6 +109,9 @@ Future<void> runInstallerApp(
     () => PageConfigService(
       config: tryGetService<ConfigService>(),
       includeTryOrInstall: includeTryOrInstall,
+      allowedToHide: InstallationStep.values
+          .where((e) => e.allowedToHide)
+          .map((e) => e.route),
```
Comment (d-loose) [inline packages/ubuntu_bootstrap/lib/installer.dart:114]:
```suggestion
          .map((e) => e.name),
```

`_excludedPages` in `PageConfigService` uses the names of the pages without the leading '/'

File: README.md [hunk 2]
```diff
@@ -82,9 +82,12 @@ theme:
 # - privacy: Enable location services
 # - timezone: Set the timezone
 # - telemetry: Enable sending telemetry
+#
+# Do note that only some non-mandatory pages offer the possibility to be hidden.
```
Comment (d-loose) [inline README.md:86]: Should we name the pages that can be hidden explicitly?

Comment (spydon) [reply README.md:86]: Right, I forgot to circle back on that, which ones do we want to be able to hide?
Are accessibility, source, try-or-install and refresh enough for stage 3?

File: packages/ubuntu_bootstrap/lib/installer.dart (hunk 1 above)
Comment (spydon) [reply packages/ubuntu_bootstrap/lib/installer.dart:114]: Wtf... How did this even work...? :sweat_smile: 
Good catch!

**Review by d-loose — APPROVED** [review]

**Pull request merged**